theforeman.org/yaml/jobs/test_3_4_stable.yaml

```diff
@@ -0,0 +1,39 @@
+- job:
+    name: test_3_4_stable
+    project-type: matrix
+    quiet-period: 2700
+    properties:
+      - github_foreman
+    scm:
+      - foreman:
+          branch: '3.4-stable'
+    triggers:
+      - scm_fifteen_minutes
+      - github
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+            - 2.5
```

Comment:
We've dropped 2.5 support for 3.4.
```suggestion
```

Reply:
Oh, nevermind.  I read this completely wrong and thought you were talking about something else.

Reply:
updated.